### DISCUSSION TITLE
'Node' object has no attribute 'get_display_name'

### DISCUSSION THREAD

**Discussion by MaxWagels** [FreeOpcUa/opcua-asyncio#470] [Q&A] [answered]
Hi,
I need to get some attributes of some nodes and have a problem with the get_display_name() function.
await client.get_node("ns=3;i=1001").get_display_name()
throws the error: AttributeError: 'Node' object has no attribute 'get_display_name'
But if i replace get_display_name() with get_parent() or get_value(), it works perfectly fine.
What am I doing wrong?
Thanks for your help

**Comment by AndreasHeine** [comment, marked as answer]
**read_display_name()**

https://github.com/FreeOpcUa/opcua-asyncio/blob/12c0e7a05d756219ac56448edf7f78a030a3d049/asyncua/common/node.py#L88

**Reply by MaxWagels** [reply]
Great, thank you.
Just another question, the output of this function is now:
LocalizedText(Encoding:2, Locale:None, Text:Signal).
How do I just extract the Text value?

**Reply by AndreasHeine** [reply]
https://github.com/FreeOpcUa/opcua-asyncio/blob/12c0e7a05d756219ac56448edf7f78a030a3d049/asyncua/ua/uatypes.py#L520

https://github.com/FreeOpcUa/opcua-asyncio/blob/12c0e7a05d756219ac56448edf7f78a030a3d049/asyncua/ua/uatypes.py#L547

".Text"

**Reply by AndreasHeine** [reply]
these are all classes which you can access like you would expect it from a python class

**Reply by MaxWagels** [reply]
Alright, thank you
